examples/can.rs

```diff
@@ -13,7 +13,9 @@ use hal::pac;
 use hal::prelude::*;
 use hal::watchdog::IndependentWatchDog;
 
-use hal::can::{Can, CanFilter, CanFrame, CanId, Filter, Frame, Receiver, Transmitter};
+use bxcan::filter::Mask32;
+use bxcan::{Frame, StandardId};
```

Comment:
Could be 
```suggestion
use can::bxcan::{Frame, StandardId};
```
if we would reexport the can module. (See other comment)